test_basic fails

Using `https://download.opensuse.org/distribution/leap/15.1/jeos/openSUSE-Leap-15.1-JeOS.x86_64-OpenStack-Cloud.qcow2` and the latest master `commit 5c75ddb2b105edaeab786c01f82bca1aa45c8a5c`, the command

```
tox -e py37 tests/test_basic.py
```
errors with

```
E               subprocess.CalledProcessError: Command '/home/rookcheck/tmp/rookcheck/rookcheck-rookcheck-a652/kubectl --kubeconfig /home/rookcheck/tmp/rookcheck/rookcheck-rookcheck-a652/kubeconfig --namespace rook-ceph exec -t "rook-direct-mount-66b7c68959-bhq9t" -- bash -c "$(cat <<'EOF'
E               
E                       # Create the directory
E                       mkdir /tmp/registry
E              
E                       # Detect the mon endpoints and the user secret for the connection
E                       mon_endpoints=$(grep mon_host /etc/ceph/ceph.conf | awk '{print $3}')
E                       my_secret=$(grep key /etc/ceph/keyring | awk '{print $3}')
E               
E                       # Mount the filesystem
E                       mount -t ceph -o mds_namespace=myfs,name=admin,secret=$my_secret             $mon_endpoints:/ /tmp/registry
E               
E                       # See your mounted filesystem
E                       df -h /tmp/registry
E               
E                       echo "Hello Rook" > /tmp/registry/hello
E                       umount /tmp/registry
E                       rmdir /tmp/registry
E     
E               EOF
E               )"' returned non-zero exit status 1.

tests/lib/common.py:171: CalledProcessError


----------------------------------- Captured log call ------------------------------------
11:35:08 kubectl apply -f /home/rookcheck/tmp/rookchec    INFO : deployment.apps/rook-direct-mount created
11:35:13 kubectl --namespace rook-ceph get pod -l app=    INFO : rook-direct-mount-66b7c68959-bhq9t
11:35:14 kubectl --namespace rook-ceph exec -t "rook-d WARNING : mount error 2 = No such file or directory
11:35:14 kubectl --namespace rook-ceph exec -t "rook-d    INFO : Filesystem      Size  Used Avail Use% Mounted on
11:35:14 kubectl --namespace rook-ceph exec -t "rook-d    INFO : overlay          10G  4.1
G  5.9G  42% /
11:35:14 kubectl --namespace rook-ceph exec -t "rook-d WARNING : umount: /tmp/registry: not mounted
11:35:14 kubectl --namespace rook-ceph exec -t "rook-d WARNING : rmdir: failed to remove '/tmp/registry': Directory not empty
11:35:14 kubectl --namespace rook-ceph exec -t "rook-d WARNING : command terminated with exit code 1
```

I'll see if I can learn anything since I can reproduce this.

--- a/tests/test_basic.py
+++ b/tests/test_basic.py
@@ -20,12 +20,23 @@ import time
 logger = logging.getLogger(__name__)
 
 
+def test_deploy_filesystem(rook_cluster):
+    # rook_cluster checks the filesystem is deployed before continuing
+    rook_cluster.deploy_rbd()
+    time.sleep(10)
+    rook_cluster.deploy_filesystem()
+    # This test leaves the filesystem set up ready for the next tests in this
+    # module.
+
+
 def test_file_creation(rook_cluster):
     logger.debug("Create direct-mount deployment")
     rook_cluster.kubernetes.kubectl_apply(
         os.path.join(rook_cluster.ceph_dir, 'direct-mount.yaml'))
 
-    time.sleep(5)
+    # TODO(jhesketh): Create a helper function for checking if a container is
+    #                 ready instead of waiting.
+    time.sleep(10)
 
     logger.debug("Mount myfs in pod and put a test string into a file")
     rook_cluster.kubernetes.execute_in_pod_by_label("""
@@ -56,12 +67,12 @@ def test_file_creation(rook_cluster):
     rook_cluster.kubernetes.kubectl(
         "scale deployment rook-direct-mount --replicas=0 -n rook-ceph")
 
-    time.sleep(5)
+    time.sleep(10)
 
     rook_cluster.kubernetes.kubectl(
         "scale deployment rook-direct-mount --replicas=1 -n rook-ceph")
 
-    time.sleep(5)
+    time.sleep(10)
 
     logger.debug("Mount myfs again and output the contents")
     rc, stdout, stderr = rook_cluster.kubernetes.execute_in_pod_by_label("""

